scripts/index.js

```diff
@@ -1 +1,159 @@
-// Your code goes here!
\ No newline at end of file
+// Positive Numbers
+// Write a function which takes an array of numbers as input and 
+// returns a new array containing only the positive 
+// numbers in the given array.
+
+function posNumbers(numbers) {
+    const positiveNums = [];
+    for (let num of numbers) {
+        if (num > 0) {
+        positiveNums.push(num);
+    }
```

Comment:
Great work! I do think you should indent this curly brace (and the one on the next line, though.